aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -26,6 +26,16 @@ TORCH_META_FUNC(_convert_indices_from_coo_to_csr) (
   set_output(size + 1, options);
 }
 
+TORCH_META_FUNC(_convert_indices_from_csr_to_coo) (
+  const Tensor& crow_indices, const Tensor& col_indices, const int64_t size, const bool out_int32
+) {
+  TORCH_CHECK(crow_indices.dim() <= 1, "crow_indices is supposed to be a vector");
+  TORCH_CHECK(col_indices.dim() <= 1, "col_indices is supposed to be a vector");
```

Comment:
What is the reason for the `<=` comparison?
```suggestion
  TORCH_CHECK(crow_indices.dim() == 1, "crow_indices is supposed to be a vector");
  TORCH_CHECK(col_indices.dim() == 1, "col_indices is supposed to be a vector");
```

Reply:
Ah. Indeed.